apps/docs/content/3-components/2-library/31-radio-button.mdx

```diff
@@ -2,13 +2,113 @@
 title: Radio Button
 description: Radio Button HTML Component
 draft: false
-status: in-development
+status: stable
 ---
 
 # Radio Button
 
 <ComponentStatusBlock componentId="radio-button" />
 
+import { standardProps } from "@/components/form/radio";
+
+## Standard radio
+
+<RadiosGroup {...standardProps} />
+
+<Tabs id="tab1">
+  <TabList>
+    <TabItem value="tab11" checked>HTML</TabItem>
+    <TabItem value="tab12">Macro</TabItem>
+    <TabItem value="tab13">React</TabItem>
+  </TabList>
+  <TabPanel value="tab11">
+  ```html
+<div class="govie-form-group "></div>
+  <legend class="govie-fieldset__legend govie-fieldset__legend--l">
```

Comment:
What are those `govie_*` classes ?

Reply:
That is the example for the local HTML library, not global one. The classes should be declared in the old repo
https://github.com/ogcio/ogcio-ds

Reply:
We have to use our global classes for that.
Check how we did for the heading or paragraph.

Reply:
fixed